smoke_test.sh

```diff
@@ -0,0 +1,90 @@
+# Spin up iqe pod and execute IQE tests in it
+
+# Env vars defined by caller:
+#IQE_PLUGINS="plugin1,plugin2" -- pytest plugins to run separated by "," #IQE_MARKER_EXPRESSION="mymarker" -- pytest marker expression
+#IQE_FILTER_EXPRESSION="something AND something_else" -- pytest filter, can be "" if no filter desired
+#NAMESPACE="mynamespace" -- namespace to deploy iqe pod into, can be set by 'deploy_ephemeral_env.sh'
+
+
+function kill_port_fwd {
+    echo "Caught signal, kill port forward"
+    if [ ! -z "$PORT_FORWARD_PID" ]; then kill $PORT_FORWARD_PID; fi
+}
+
+# The CJI var name will need to be exported in the main pr_check.sh
+oc apply -n $NAMESPACE -f $APP_ROOT/$CJI_PATH
```

Comment:
Perhaps a bonfire command should be written that applies a CJI template for you. Actually, it could probably take care of all the 'wait for job to appear', 'wait for it to run', etc. etc. logic you have below as well (less bash looping :P). And then also if we make small tweaks to the spec of a CJI we won't have to tell 50 repos to update their CJI -- we could just update the template in bonfire